python/paddle/tensor/manipulation.py

```diff
@@ -5166,3 +5166,104 @@ def unfold(x, axis, size, step, name=None):
 }
 for name, func in __METHODS.items():
     setattr(core.eager.Tensor, name, func)
+
+
+def _index_fill_impl(x, index, axis, value, inplace):
+    if not isinstance(index, Variable):
+        raise ValueError("index must be Tensor")
+
+    if not isinstance(value, Variable):
+        value = paddle.to_tensor(value, dtype=x.dtype)
+    else:
+        if len(value.shape) > 0:
+            raise ValueError("value must be scalar or 0-D tensor")
+
+    x_dim = len(x.shape)
+    if axis < 0:
+        axis = axis + x_dim
+
+    if not (isinstance(axis, int)) or (axis > x_dim - 1) or axis < -x_dim:
```

Comment:
The negative `axis` has been processed in L5183 above, so the judgment condition here should be `axis < 0` not `axis < -x_dim`

Reply:
Done